### ISSUE TITLE
Incompatible types between PragmaRX\Countries\Package\Services\Cache\Service and Psr\SimpleCache\CacheInterface

### ISSUE THREAD

**Issue by ghunti** [antonioribeiro/countries#207]
Hi @antonioribeiro 

The latest version of thhis package, changed the compatiblity of the `psr/simple-cache` to ^3.0.
The problem is that now there's an incompatibility between PragmaRX\Countries\Package\Services\Cache\Service and Psr\SimpleCache\CacheInterface:

```
Declaration of PragmaRX\Countries\Package\Services\Cache\Service::get($ke  
  y,                                                                           
     $default = null) must be compatible with Psr\SimpleCache\CacheInterface:  
  :g                                                                           
    et(string $key, mixed $default = null): mixed 
```

**Comment by antonioribeiro** [comment]
Yes, sorry about that, I believe I'll have to tag a major version on this one as it's a breaking change. Are you able to use a fixed version (previous one) for now?

**Comment by ghunti** [comment]
Hello @antonioribeiro ? :-)

**Comment by ghunti** [comment]
Hey @antonioribeiro I think #203 fixes the issue

**Comment by ghunti** [comment]
Hey @antonioribeiro totally understand X-) Can you point me to the PR/branch where you are working? I might be able to help :-)

**Comment by VictorMGomes** [comment]
Hello i'm facing the same problem:

[2024-12-21 22:52:26] local.ERROR: Declaration of PragmaRX\Countries\Package\Services\Cache\Service::get($key, $default = null) must be compatible with Psr\SimpleCache\CacheInterface::get(string $key, mixed $default = null): mixed {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Declaration of PragmaRX\\Countries\\Package\\Services\\Cache\\Service::get($key, $default = null) must be compatible with Psr\\SimpleCache\\CacheInterface::get(string $key, mixed $default = null): mixed at /var/www/html/vendor/pragmarx/countries/src/package/Services/Cache/Service.php:90)
[stacktrace]

For now i back this package to 0.8.2

**Comment by AlexKomMsk** [comment]
Та же самая проблема. И откатить версию не могу
